docs/UserGuide.adoc

```diff
@@ -775,25 +776,28 @@ image::ui-screenshots/uimockup1_online.png[@Online Tab View, 600, 400]
 .Mark's Offline Tab View
 image::ui-screenshots/uimockup1_offline.png[@Offline Tab View, 600, 400]
 
+_{ more coming soon }_
 
-*What is the [[Dashboard]] Dashboard?*
+== Glossary
```

Comment:
Maybe we can reorder the glossary? Like _Bookmark_ -> _Offline-copy_ -> _Annotation_ -> _Folder Hierarchy_ -> _Dashboard_. Since some of the definition depends on other definition.

Reply:
I did consider that, but I think it's okay because the glossary isn't meant to be read in order. It's currently arranged in alphabetical order so that the user can easily find a word/phrase that they're looking for.

Reply:
Okay